app/controllers/rooms_controller.rb

```diff
@@ -115,41 +115,65 @@ def game_player_count_post
     count = params[:player_count].to_i
     @room.player_count = count
     @room.save
+    # Delete players that no longer fit in this game
+    ActiveRecord::Base.connection.execute("DELETE FROM room_players where room_id = #{@room.id} AND player_number > #{(count / 2).to_i}")
```

Comment:
I don't think so. I think that would happen if you used `#destroy_all` instead of `#delete_all`, but I could be wrong. I can fire up rails c and check if needed.